systemtests/test_env.py

```diff
@@ -89,56 +88,71 @@ def test_env(ctx, env):
             ctx.invoke(
                 upload_symbols,
                 expect_code=201,
-                auth_token=f"{auth_token}",
-                base_url=f"{tecken_host}",
+                auth_token=auth_token,
+                base_url=tecken_host,
                 symbolsfile=f"{ZIPS_DIR}{fn}",
             )
+        click.echo("")
 
         click.echo(
             click.style(">>> UPLOAD BY DOWNLOAD TEST (DESTRUCTIVE)", fg="yellow")
         )
-        URL = ctx.invoke(list_firefox_symbols_zips, number=1, max_size=1000000000)[0]
+        url = ctx.invoke(list_firefox_symbols_zips, number=1, max_size=1000000000)[0]
         ctx.invoke(
             upload_symbols_by_download,
-            base_url=f"{tecken_host}",
-            auth_token=f"{auth_token}",
-            url=f"{URL}",
+            base_url=tecken_host,
+            auth_token=auth_token,
+            url=url,
```

Comment:
Ah I guess we don't need an f-string if we're just passing the variable directly here, and it's not a substring for something else.